test/test-controller.ts

```diff
@@ -92,73 +92,92 @@ describe('Controller API', function() {
 
     // register before each test
     before(function(done) {
-      nock(url)
-        .post(api + '/debuggees/register')
-        .reply(200, {
-          debuggee: { id: 'fake-debuggee' },
-          activePeriodSec: 600
-        });
+      nock(url).post(api + '/debuggees/register').reply(200, {
+        debuggee: {id: 'fake-debuggee'},
+        activePeriodSec: 600
+      });
       const debuggee = new Debuggee({
         project: 'fake-project',
         uniquifier: 'fake-id',
         description: 'unit test'
       });
       const controller = new Controller(fakeDebug);
-      controller.register(debuggee, function(err/*, result*/) {
+      controller.register(debuggee, function(err /*, result*/) {
         assert.ifError(err);
         done();
       });
     });
 
     it('should deal with a missing breakpoints response', function(done) {
-      const scope = nock(url)
-        .get(api + '/debuggees/fake-debuggee/breakpoints?successOnTimeout=true')
-        .reply(200, { kind: 'whatever' });
+      const scope =
+          nock(url)
+              .get(
+                  api +
```

Comment:
Nit. Does the lint move 'api +' in a new line here and elsewhere?

Reply:
`gts fix` made this change.